head.html

```diff
@@ -24,8 +24,49 @@
     if (footer.innerHTML == "/footer.plain.html") footer.innerHTML = "";
   }
 
+  function getSection(index) {
+    const nodes = document.querySelectorAll("main div");
+    return nodes.length > index ? nodes[index] : null;
+  }
+
+  function fetchAuthor() {
+    const div = getSection(2);
+    if (div) {
+      const r = /by (.*)\n*(.*)$/gm.exec(div.innerText);
```

Comment:
```suggestion
      const r = /^by (.*)$/gm.exec(div.innerText);
```
Wouldn't this be more concise?

Reply:
Ah, I can see now 😄 you need the second match for the date... then how about:
```suggestion
      const r = /^[Bb]y (.*)\n*(.*)$/gm.exec(div.innerText);
```
Word tends to auto-uppercase the first character in a sentence or paragraph. And you want to make sure the "By" is at the beginning of the new line.